index.bs

```diff
@@ -2632,10 +2632,11 @@ run the following steps:</p>
 
       <ol>
 
-       <li><p>If |value| contains a U+002C COMMA character (,), then let |linepos| be the leading
-       substring of |value| up to and excluding the first U+002C COMMA character (,) in that string
-       and let |linealign| be the trailing substring of |value| starting from the character
-       immediately after the first U+002C COMMA character (,) in that string.</p></li>
+       <li><p>If |value| contains a U+002C COMMA character (,) that is not the last character in
```

Comment:
I wanted `line:50,` to be dropped. But maybe can set linealign to null instead of empty string, and check for that, instead.